StardewArchipelago/ModEntry.cs

```diff
@@ -447,7 +447,7 @@ private void OnDayStarted(object sender, DayStartedEventArgs e)
             Helper.GameContent.InvalidateCache("Data/Shops"); // This should be reworked someday
             DoBugsCleanup();
 
-            _hintHelper.GiveHintTip(_archipelago.Session);
+            _hintHelper?.GiveHintTip(_archipelago.Session);

```

Comment:
Is that one ever null?

Reply:
What I left it ? Sorry, it just kept erroring for some reason and somehow missed it while I was cleaning up everything

Reply:
You might have been working on a version that had some of the init bugs that I fixed in 6.2.0